contents/handbook/engineering/revenue-and-forecasting.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Revenue and forecasting
+sidebar: Handbook
+showTitle: true
+---
+
+The growth team maintains the revenue dashboards and queries that are used to understand:
+
+1. What our historical revenue record looks like
+2. What our revenue is expected to be this month
+3. What our churn, growth, expansion, and contraction look like
+4. Which customers have done the above activities
+5. etc
+
+Currently, all revenue dashboards can be found in Metabase (though we hope to have them all in PostHog's own data warehouse soon 👀).
+
+## Important dashboards
+(these require internal access)
+- [General overview](http://metabase-prod-us/dashboard/1-overview): Useful to see our ARR graph and get quick links to dig into revenue for certain months.
+- [Revenue lifecycle](http://metabase-prod-us/dashboard/38-revenue-lifecycle-deep-dive-dashboard): Useful to see trends over time for churn, expansion, new revenue, etc.
+- [Product-specific analysis](http://metabase-prod-us/dashboard/39-revenue-growth-by-product (pick your product and date [month] at the top!)): Kind of a combination of the above two, but for a specific product
+- [Revenue by customer bucket](http://metabase-prod-us/dashboard/40-revenue-growth-by-customer-spend-size): I've yet to find this dashboard very useful, but maybe that time hasn't come yet, or maybe you will dig up some gold there :)
```

Comment:
I find this dashboard super useful from a financial modelling and board reporting perspective! (So at least 1 person is using it regularly 😄)

Reply:
Good to know!

```suggestion
- [Revenue by customer bucket](http://metabase-prod-us/dashboard/40-revenue-growth-by-customer-spend-size): Useful for understanding trends in contract size for financial modeling, support, etc
```